documentation/modules/cruise-control/con-optimization-proposals.adoc

```diff
@@ -94,10 +101,45 @@ m¦excludedBrokersForReplicaMove
 
 |===
 
+
+[discrete]
+=== Broker load
+
+The broker load is a json string which consist of all broker IDs and provides a number of metrics for each broker.
+Each metric consist of three values.
+The first is the metric value before the optimization proposal is applied, the second is the expected value of the metric after the proposal is applied, and the third is the difference between the first two values (after minus before).
+
+The following table explains the properties contained in the optimization proposal's broker load section:
```

Comment:
"The following table explains the properties contained in the optimization proposal's broker load ConfigMap"